config/runtime.exs

```diff
@@ -23,7 +23,8 @@ if config_env() != :test do
     message_log_s3_bucket: System.get_env("MESSAGE_LOG_S3_BUCKET"),
     message_log_s3_folder: System.get_env("MESSAGE_LOG_S3_FOLDER"),
     message_log_report_s3_folder: System.get_env("MESSAGE_LOG_REPORT_S3_FOLDER"),
-    monitoring_api_key: System.get_env("MONITORING_API_KEY")
+    monitoring_api_key: System.get_env("MONITORING_API_KEY"),
+    active_headend_path: System.get_env("ACTIVE_HEADEND_S3_PATH", "/headend.json")
```

Comment:
Assuming this is set correctly in all environments, I think it would be better to avoid having a default value.